### PR TITLE
feat: Add keymap descriptions

### PR DESCRIPTION
**Pull request by rafamadriz**
Let me know if you're happy with the descriptions I added, here's a screenshot of the final result:

Screenshot_20220802_212934

### CODE REVIEW

**Comment by numToStr** [comment]
Thanks @rafamadriz for putting this together.

Just one thing, isn't it better to start the description with `Toggle comment ...`. Thoughts?

**Comment by rafamadriz** [comment]
@numToStr I thought about that, but I felt that it was better to have a naming convention where you first start with `name/label` -> `action`. So for example, you have `Comment` and `Uncomment` as names/labels and then action to happen. But besides that, I don't really have a good reason to not do it the other way, just let me know if you want me to change it

File: lua/Comment/api.lua [hunk 1]
```diff
@@ -248,44 +248,109 @@ function api.setup(config)
...
+            )
 
             -- VISUAL mode extended
-            K('x', 'g>', '<ESC><CMD>lua require("Comment.api").locked.comment_linewise_op(vim.fn.visualmode())<CR>')
-            K('x', 'g<', '<ESC><CMD>lua require("Comment.api").locked.uncomment_linewise_op(vim.fn.visualmode())<CR>')
+            K(
+                'x',
+                'g>',
+                '<ESC><CMD>lua require("Comment.api").locked.comment_linewise_op(vim.fn.visualmode())<CR>',
+                { desc = 'Comment region using single line (visual)' }
```
Comment (numToStr) [inline lua/Comment/api.lua:346]:
```suggestion
                { desc = 'Comment region linewise (visual)' }
```

File: lua/Comment/api.lua [hunk 2]
```diff
@@ -248,44 +248,109 @@ function api.setup(config)
...
+            )
+            K(
+                'x',
+                'g<',
+                '<ESC><CMD>lua require("Comment.api").locked.uncomment_linewise_op(vim.fn.visualmode())<CR>',
+                { desc = 'Uncomment region using single line (visual)' }
```
Comment (numToStr) [inline lua/Comment/api.lua:352]:
```suggestion
                { desc = 'Uncomment region linewise (visual)' }
```

**Review by numToStr — CHANGES_REQUESTED** [review]

**Comment by numToStr** [comment]
@rafamadriz I agree with you. Please review the requested changes then we could go ahead with merging this :)

**Review by numToStr — APPROVED** [review]
Thanks for working on this 👍🏻

**Pull request merged**